fe/fe-core/src/main/java/org/apache/doris/datasource/InternalDataSource.java

```diff
@@ -1703,11 +1703,6 @@ private void createOlapTable(Database db, CreateTableStmt stmt) throws UserExcep
         OlapTable olapTable = new OlapTable(tableId, tableName, baseSchema, keysType, partitionInfo,
                 defaultDistributionInfo, indexes);
 
-        for (Column column : baseSchema) {
```

Comment:
We need to ensure that this change won't break the original behavior of base table.